test/unit/context.ts

```diff
@@ -59,6 +59,19 @@ describe('Context', () => {
       expect(plugins).to.have.members(['local']);
     });
 
+    describe('hook handlers with only one argument', () => {
+      it('are passed the "done" callback function instead of the argument passed to emitHook', async() => {
```

Comment:
Is this a test that's documenting incorrect, buggy behavior? If so, please add a comment to that effect, or better yet, add it to the test name.

Did you take a look at how hard it would be to fix it instead?